### PR TITLE
Add config section

### PR DESCRIPTION
**Pull request by m1**
@josegonzalez as per req

### CODE REVIEW

**Comment by jadb** [comment]
This isn't just a configuration section, these are all 'new' packages to be listed so might take a little time to check them all out. Maybe submit them separately? 

I haven't checked them all yet but I vouch for @josegonzalez's package as I use it on a daily basis. 

The php-wise package should be completely removed (only one that was on the list already) because it has been abandoned *3 years ago!*

**Comment by m1** [comment]
@jadb lol what? It doesn't take long to check a package, like 5 seconds maybe. You're right about php-wise though.

File: README.md [hunk 1]
```diff
@@ -130,6 +131,14 @@ Please see [CONTRIBUTING](https://github.com/ziadoz/awesome-php/blob/master/CONT
 * [Hoa Project](http://hoa-project.net/En/) - Another package of PHP components.
 * [League of Extraordinary Packages](https://thephpleague.com/) - A PHP package development group.
 
+## Configuration
+*Libraries for loading configuration files*
+
+* [PHP dotenv](https://github.com/josegonzalez/php-dotenv) - Loads environment variables from `.env`..
+* [Vars](https://github.com/m1/vars) - Loads config variables from env, ini, json, php, toml, xml, yml. Supports custom loaders
+* [Yaml](https://github.com/symfony/yaml) - YAML file loader
+* [Env](https://github.com/m1/env) - Env file loader
```
Comment (josegonzalez) [inline README.md]: Technically it's an env file "parser". Doesn't load anything :P

**Comment by jadb** [comment]
@m1 please read the [Contributing guidelines](https://github.com/ziadoz/awesome-php/blob/master/CONTRIBUTING.md).

Closing this as it does not adhere to the guidelines and I suggested making changes already.

**Pull request closed without merging**